### ISSUE TITLE
Welcome to Lantern Pro screen is shown to the user after authorizing a device

### ISSUE THREAD

**Issue by sunshine19090** [getlantern/lantern#4759]
It should only have a message related to authorizing the device.

<img width="1280" alt="screen shot 2016-06-21 at 4 20 56 pm" src="https://cloud.githubusercontent.com/assets/11394369/16249635/38497d10-37cc-11e6-9c32-d29e09bd9cf2.png">

**Comment by p3rcya** [comment]
I think the screen should be kept to remind them about the invite. We should however, change the title of the wording to "This device is using Pro now. Invite your friends to get more free Pro months." @chipstehr

**Comment by sunshine19090** [comment]
@p3rcya I agree on the invite reminder but we should also provide them the ability to authorize the device again. 

I would suggest to show the free user UI. The user could use the login feature or authorize the device in case they want it back. But we may have to show the referral code of the pro user in the free user UI if that is possible.

**Comment by p3rcya** [comment]
What do you mean by "provide them the ability to authorize the device again"? Didn't the user just authorize the very device?

**Comment by sunshine19090** [comment]
Sorry @p3rcya , I deleted my previous comment as I mistook your comment for another ticket.

**Comment by chipstehr** [comment]
Percy you just want the title to change or the whole thing? 

# You are now using Pro! 
Invite your friends and get free months of Pro! 
Get Free Credits [button that takes you to referral screen]

**Comment by p3rcya** [comment]
@Chip, your wording is good! Let use "You are now using Pro!"

On Wednesday, June 22, 2016, Chip <notifications@github.com> wrote:

> Percy you just want the title to change or the whole thing?
> You are now using Pro!
>
> Invite your friends and get free months of Pro!
> Get Free Credits [button that takes you to referral screen]
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern/issues/4759#issuecomment-227759470>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHjwqVxsidkRbTMJG8S6PJZFTlZhzad5ks5qOUUXgaJpZM4I7RQf>
> .
>


-- 
Percy Alpha(PGP
<https://pgp.mit.edu/pks/lookup?op=vindex&search=0xF30D100F7FE124AE>)

**Comment by chipstehr** [comment]
@p3rcya But one issue is that we invite people to invite friends but don't give them the button to click to the referral page...

**Comment by p3rcya** [comment]
After the user click continue, he will be taken to the Pro main page, which has the referral. So I think it's fine. But ideally, we should even add a small referral code in this screen too @Derekf5

**Comment by chipstehr** [comment]
Yea just think it should be direct if that's what we want them to do. What do we get if they go to the Pro page?

**Comment by chipstehr** [comment]
Although we probably don't want to mess with UI at this point tbh

**Comment by p3rcya** [comment]
@chipstehr We can discuss adding the referral code screen after MVP

**Comment by atavism** [comment]
@myleshorton I noticed this one too trying to authorize desktop for Pro (after purchasing Pro on mobile).

**Comment by xiam** [comment]
Not sure if I understand why this is a bug, if you're authorizing a device
and succeed you become pro. Isn't that always true?

**Comment by atavism** [comment]
@xiam Right, but you're authorizing a device, not making another purchase or signing up for Pro for the first time. So seeing the welcome message doesn't make sense

**Comment by Derekf5** [comment]
Sorry i missed this ticket, seems to me the thing to do would just be to make this notification a snackbar for those who have just authorized pro.   We dont need the pop up to remind users to invite, because that is the full main page when a user signs in to pro.

**Comment by Derekf5** [comment]
This shouldnt be a blocker for the MVP tho, if it is easier right now to just change the language to what @chipstehr suggested, i would go that route.
